testing/dart/color_filter_test.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:ui';
 
 import 'package:litetest/litetest.dart';
 
+const Color transparent = Color(0x00000000);
```

Comment:
Uber it: transparentBlack

Reply:
If there's a test failure I have to amend it for...